js_modules/dagit/packages/core/src/assets/PartitionHealthSummary.tsx

```diff
@@ -144,9 +146,9 @@ export const PartitionHealthSummary: React.FC<{
             style={{
               left: indexToPct(s.startIdx),
```

Comment:
I wonder, could we also try a quick test with like 10,000 indexes? It's possible that this left value could be 99.9999% for the last item and even with a min-width it'd be hidden off the edge. Maybe we could write this as `left: "calc(max(100% - 2px, ${indexToPct(s.startIdx)}"` and let CSS deal with it... :-/

Reply:
good call. Found and fixed.